concertfit/Controllers/HomeController.cs

```diff
@@ -11,7 +11,8 @@
 
 namespace concertfit.Controllers
 {
-    [Route("api")]
+    [Route("TicketMaster")]
+    [BindProperties(SupportsGet = true)]
```

Comment:
Interesting catch here... .Net automatically binds properties on a POST, but not on a GET, as POST is usually a write operation, while GET is a read operation.  So you have to add this flag to get data passed in on a GET.